node/node.go

```diff
@@ -198,7 +198,7 @@ func CustomReactors(reactors map[string]p2p.Reactor) Option {
 				for _, chDesc := range reactor.GetChannels() {
 					if !ni.HasChannel(chDesc.ID) {
 						ni.Channels = append(ni.Channels, chDesc.ID)
-						n.transport.AddChannel(chDesc.ID)
+						n.transport.AddChannel(chDesc.ID) //nolint:errcheck
```

Comment:
How about logging in with debug level?

Reply:
Sorry, It was nodeinfo with different fields, so logging is still necessary.